autoconfigure/storage-logzio/pom.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.logz.zipkin</groupId>
+        <artifactId>zipkin-logzio-autoconfigure</artifactId>
+        <!--suppress MavenPropertyInParent -->
+        <version>${zipkin-logzio-version}</version>
+    </parent>
+
+    <artifactId>zipkin-autoconfigure-storage-logzio</artifactId>
+
+    <properties>
+        <main.basedir>${project.basedir}/../..</main.basedir>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.logz.zipkin</groupId>
+            <artifactId>zipkin-storage-logzio</artifactId>
+            <version>${zipkin-logzio-version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.google.guava</groupId>
```

Comment:
zipkin-server already includes a version of guava, and guava causes an alarming amount of version drift bugs. Better to exclude it as I did here.